modules/apps/portal/portal-default-permissions-web/src/main/java/com/liferay/portal/defaultpermissions/web/internal/configuration/display/PortalDefaultPermissionsGroupConfigurationScreenWrapper.java

```diff
@@ -0,0 +1,110 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.portal.defaultpermissions.web.internal.configuration.display;
+
+import com.liferay.configuration.admin.display.ConfigurationScreen;
+import com.liferay.configuration.admin.display.ConfigurationScreenWrapper;
+import com.liferay.portal.defaultpermissions.resource.PortalDefaultPermissionsModelResourceRegistry;
+import com.liferay.portal.defaultpermissions.web.internal.display.context.PortalDefaultPermissionsConfigurationDisplayContext;
+import com.liferay.portal.defaultpermissions.web.internal.display.context.PortalDefaultPermissionsGroupConfigurationDisplayContext;
+import com.liferay.portal.kernel.feature.flag.FeatureFlagManagerUtil;
+import com.liferay.portal.kernel.language.Language;
+import com.liferay.portal.kernel.model.Group;
+import com.liferay.site.settings.configuration.admin.display.SiteSettingsConfigurationScreenContributor;
+import com.liferay.site.settings.configuration.admin.display.SiteSettingsConfigurationScreenFactory;
+
+import java.util.Locale;
+
+import javax.servlet.ServletContext;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Stefano Motta
+ */
+@Component(service = ConfigurationScreen.class)
+public class PortalDefaultPermissionsGroupConfigurationScreenWrapper
+	extends ConfigurationScreenWrapper {
+
+	@Override
+	protected ConfigurationScreen getConfigurationScreen() {
+		return _siteSettingsConfigurationScreenFactory.create(
+			new DefaultPermissionsSiteSettingsConfigurationScreenContributor());
+	}
+
+	@Reference
+	private Language _language;
+
+	@Reference
+	private PortalDefaultPermissionsModelResourceRegistry
+		_portalDefaultPermissionsModelResourceRegistry;
+
+	@Reference(
+		target = "(osgi.web.symbolicname=com.liferay.portal.defaultpermissions.web)"
+	)
+	private ServletContext _servletContext;
+
+	@Reference
+	private SiteSettingsConfigurationScreenFactory
+		_siteSettingsConfigurationScreenFactory;
+
+	private class DefaultPermissionsSiteSettingsConfigurationScreenContributor
+		implements SiteSettingsConfigurationScreenContributor {
+
+		@Override
+		public String getCategoryKey() {
+			return "default-permissions";
+		}
+
+		@Override
+		public String getJspPath() {
+			return "/configuration" +
+				"/portal_default_permissions_configuration.jsp";
+		}
+
+		@Override
+		public String getKey() {
+			return "portal-default-permissions-group-configuration";
```

Comment:
this key

Reply:
this one si correct that is different because it's a different screenwrapper that use the same jsp